libs/gi/ui/src/components/artifact/ArtifactFilterDisplay.tsx

```diff
@@ -226,33 +244,6 @@ export function ArtifactFilterDisplay({
                 filterOptionDispatch({ artSetKeys })
               }
             />
-          </Stack>
-        </Box>
-        {/* Sub-Stats */}
-        <Box p={1}>
-          <Trans t={t} i18nKey="subheadings.substats" />
```

Comment:
Can you remove the corresponding translation at https://github.com/frzyc/genshin-optimizer/blob/master/libs/gi/localization/assets/locales/en/artifact.json#L110 ?

Reply:
Done